### ISSUE TITLE
[json.exception.parse_error.101] parse error at line 1, column 299: syntax error while parsing value - unexpected '{'; expected 

### ISSUE THREAD

**Issue by pschonmann** [Fushko/gummy#15]
Hi, when i run gummy in loop, ive got this error. Not all time but very often.

```
for i in $(seq 0 3); do gummy -b 100 -s $i;done
terminate called after throwing an instance of 'nlohmann::detail::parse_error'
  what():  [json.exception.parse_error.101] parse error at line 1, column 299: syntax error while parsing value - unexpected '{'; expected end of input
```

Then gummyd dies

syslog

```
Aug 14 12:21:11 E495 systemd-coredump[217746]: Process 215074 (gummyd) of user 1000 dumped core.#012#012Stack trace of thread 215074:#012#0  0x00007fbd15a0500b __GI_raise (libc.so.6 + 0x4300b)#012#1  0x00007fbd159e4859 __GI_abort (libc.so.6 + 0x22859)#012#2  0x00007fbd15dbc911 n/a (libstdc++.so.6 + 0x9e911)#012#3  0x00007fbd15dc838c n/a (libstdc++.so.6 + 0xaa38c)#012#4  0x00007fbd15dc83f7 _ZSt9terminatev (libstdc++.so.6 + 0xaa3f7)#012#5  0x00007fbd15dc86a9 __cxa_throw (libstdc++.so.6 + 0xaa6a9)#012#6  0x00005600218693fa _ZN8nlohmann6detail28json_sax_dom_callback_parserINS_10basic_jsonISt3mapSt6vectorNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEblmdSaNS_14adl_serializerES4_IhSaIhEEEEE11parse_errorINS0_11parse_errorEEEbmRKSA_RKT_.isra.0.part.0 (gummyd + 0x63fa)#012#7  0x000056002188b722 _ZN8nlohmann6detail6parserINS_10basic_jsonISt3mapSt6vectorNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEEblmdSaNS_14adl_serializerES4_IhSaIhEEEENS0_22iterator_input_adapterIN9__gnu_cxx17__normal_iteratorIPKcSA_EEEEE5parseEbRSE_ (gummyd + 0x28722)#012#8  0x0000560021872592 _ZN7MessageC1ERKNSt7__cxx1112basic_stringIcSt11char_traitsIcESaIcEEE (gummyd + 0xf592)#012#9  0x000056002188c474 _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x29474)#012#10 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#11 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#12 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#13 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#14 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#15 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#16 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#17 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#18 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#19 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#20 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#21 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#22 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#23 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#24 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#25 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#26 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#27 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#28 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#29 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#30 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#31 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#32 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#33 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#34 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#35 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#36 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#37 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#38 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#39 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#40 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#41 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#42 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#43 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#44 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#45 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#46 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#47 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#48 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#49 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#50 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#51 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#52 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#53 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#54 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#55 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#56 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#57 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#58 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#59 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#60 0x000056002188c4df _Z12message_loopR4XorgRN4core18Brightness_ManagerERNS1_12Temp_ManagerE (gummyd + 0x294df)#012#61 0x000056002186d58f main (gummyd + 0xa58f)#012#62 0x00007fbd159e6083 __libc_start_main (libc.so.6 + 0x24083)#012#63 0x000056002186d95e _start (gummyd + 0xa95e)#012#012Stack trace of thread 215077:#012#0  0x00007fbd15f0f376 futex_wait_cancelable (libpthread.so.0 + 0xf376)#012#1  0x00007fbd15deee30 _ZNSt18condition_variable4waitERSt11unique_lockISt5mutexE (libstdc++.so.6 + 0xd0e30)#012#2  0x000056002188f7ee _ZN4core10temp_startERNS_12Temp_ManagerE (gummyd + 0x2c7ee)#012#3  0x00007fbd15df4de4 n/a (libstdc++.so.6 + 0xd6de4)#012#4  0x00007fbd15f08609 start_thread (libpthread.so.0 + 0x8609)#012#5  0x00007fbd15ae1133 __clone (libc.so.6 + 0x11f133)#012#012Stack trace of thread 215075:#012#0  0x00007fbd15f0f7d1 futex_abstimed_wait_cancelable (libpthread.so.0 + 0xf7d1)#012#1  0x000056002188eb62 _ZN4core13Gamma_Refresh4loopER4Xorg (gummyd + 0x2bb62)#012#2  0x000056002188eb7d _ZN4core1
```

**Comment by Fushko** [comment]
- How many screens do you have connected?
- What does your `~/.config/gummyconf` look like?

**Comment by Fushko** [comment]
Fixed in 0.3
